examples/example-with-encryption/packer/README.md

```diff
@@ -0,0 +1,16 @@
+# Example TLS Certificates
```

Comment:
You may want to mention how you created these so users know how to create their own!